### ISSUE TITLE
Species translation for unknown

### ISSUE THREAD

**Issue by MelinaHoule** [edwardsmarc/CASFRI#470]
Unknown species can be expressed different ways :
- unclassified species
- unkown species
- other species (not mentionned in the formal list given for a specific inventory)

CASFRI used to have 2 unknown categories:  Unkown (UNKN_SPP_###) and unclassified (UNCL_SPP_###). Do we keep those 2 values? or do we mark them all as unknown (UNKN_SPP_###). What should we do with other species (do we mark them as unknown)?

I propose we drop unclassified and we mark them and the other species value as unknown (UNKN_SPP_###).

**Comment by edwardsmarc** [comment]
I agree.

"What should we do with other species" - are you referring to the codes we don't have a translation for? I guess we can label them UNKN_SPP_### and update the value layer if we find out what they are?

**Comment by MelinaHoule** [comment]
No, "Other species" is a category found for example in BC. The definition is: Other tree, not on list . 

Other provinces use that category to emphasize that species within the polygon are species not covered by the inventory. So it is a little bit more information than only unknown. I don't know if it is relevant to keep that information at the national level because "not in list" in BC doesn't mean "not in list" in QC. Right now, they are UNKN_SPP_###. Should we change them to something else to indicate they are unknown not from the province list???

**Comment by edwardsmarc** [comment]
I see, so the choice for the BC code is either UNKN_SPP_### or UNKN_SPP_BC_# for example?

I agree that at the national scale it is simply an unknown species code. So I would use UNKN_SPP_###.

**Comment by pedrogit** [comment]
Was something done about this? Can we close it?
